<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QWidget" name="info" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout" columnminimumwidth="50,50,600">
         <item row="1" column="1">
          <widget class="QPushButton" name="StopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QLed" name="LED" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QGroupBox" name="AddOnInfo">
           <property name="title">
            <string>AddOn</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="StartButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,4,1,5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Experiment ID</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ExperimentID"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>UDP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="UDP"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="viewer">
       <attribute name="title">
        <string>Viewer</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>731</width>
          <height>621</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="settingGUI" stretch="1,5">
         <item>
          <widget class="QGroupBox" name="settingBox">
           <property name="title">
            <string>Setting</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>20</y>
              <width>711</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="settingLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>File directory</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="SettingDir"/>
             </item>
             <item>
              <widget class="QPushButton" name="OpenSettingButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LoadSettingButton">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="Hello">
           <property name="title">
            <string>Hello :)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLed</class>
   <extends>QWidget</extends>
   <header>QLed</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
